### ISSUE TITLE
BUG - Sev 3 - All - Screen Reader Reads Incorrect Phone Number on Error

### ISSUE THREAD

**Issue by TimRoe** [department-of-veterans-affairs/va-mobile-app#7865]
## What happened?

Bug was not directly observed, but discovered looking through code for researching an unrelated issue.

The problematic code (`src/components/CommonErrorComponents/CallHelpCenter.tsx`):
```
<ClickToCallPhoneNumber
  a11yLabel={a11yLabelID(t('8006982411'))}
  displayedText={callPhone ? undefined : displayedTextPhoneNumber(t('8006982411'))}
  phone={callPhone ? callPhone : t('8006982411')}
/>
```
where note that if `callPhone` is set that the a11yLabel is still set to read the default phone number for the screen reader.

## Specs:

- Accessibility State: Screen reader turned on

## Steps to Reproduce

Did not directly produce bug, but conferred with FE engineers in slack to confirm expectation of bugged behavior exists and should be observed within certain errors arising from notable API failures resulting in a redirect to call the for help VA instead.

## Desired behavior

The screen reader should read the phone number that will be called.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: Low 
- Frequency: Low 

- 3 - Low

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [x] Steps to reproduce are defined
- [x] Desired behavior is added
- [x] Labels added (front-end, back-end, relevant feature, accessibility, etc)
- [x] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the [bug epic](https://app.zenhub.com/workspaces/va-mobile-product-view-610035bc5395bb000e62e529/issues/gh/department-of-veterans-affairs/va-mobile-app/5367)

**Comment by TKDickson** [comment]
Not currently looking at this (falls outside the "3 ticket limit"), so adding blocked label.

**Comment by TKDickson** [comment]
Removing blocked, this is one of the three I'll work on

**Comment by TKDickson** [comment]
Does what it says on the tin - approved to merge.

**Comment by TKDickson** [comment]
Doesn't require additional testing in develop, closing.
